### PR TITLE
remove whenever gem, add heroku scheduler file

### PR DESCRIPTION
**Pull request by cdale77**
Moving from whenever gem, which does not work with Heroku, to a heroku scheduler file

### CODE REVIEW

File: lib/tasks/scheduler.rake [hunk 1]
```diff
@@ -0,0 +1,5 @@
+"Heroku scheduler file"
```
Comment (gkop) [inline lib/tasks/scheduler.rake]: Should this be here? Or a comment instead?

Comment (cdale77) [inline lib/tasks/scheduler.rake]: stupid mistake, missing a desc keyword

File: lib/tasks/scheduler.rake [hunk 2]
```diff
@@ -0,0 +1,5 @@
+desc "Heroku scheduler file"
+
+task get_bridge_troll_class_count: :environment do
```
Comment (gkop) [inline lib/tasks/scheduler.rake]: The argument to `desc` should describe the following `task`, and there should be no newline between the methods.

**Comment by cdale77** [comment]
Adding a comment to remind myself that after merging I need to set the rake task via heroku gui

**Comment by btaitelb** [comment]
+1

**Pull request merged**